nncf/experimental/tensorflow/graph/model_transformer.py

```diff
@@ -0,0 +1,59 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from nncf.common.graph.model_transformer import ModelTransformer
+from nncf.common.graph.transformations.commands import TransformationType
+from nncf.experimental.tensorflow.nncf_network import NNCFNetwork
+from nncf.experimental.tensorflow.graph.transformations.layout import TFTransformationLayoutV2
+from nncf.experimental.tensorflow.patch_tf import TFPatcher
+from nncf.experimental.tensorflow.patch_tf import Hook
+
+
+class TFModelTransformerV2(ModelTransformer):
```

Comment:
"V2" isn't very informative. Either change the name, or add a docstring that elaborates on how "V2" is actually different from the regular transformer.

Reply:
Thanks for the comment! I added an additional explanation to the docstring.